features/networking/egress-ip.feature

```diff
@@ -50,12 +43,19 @@ Feature: Egress IP related features
     Then the step should succeed
     Given 2 pods become ready with labels:
       | name=test-pods |
+    And evaluation of `pod(0).name` is stored in the :pod1 clipboard
+    And evaluation of `pod(1).name` is stored in the :pod2 clipboard
+
+    # add the egress ip to the hostsubnet
+    And the valid egress IP is added to the node
+
 
     # add the egress ip to the project
     When I run the :patch admin command with:
```

Comment:
Same as earlier comments. Try to shorten 54-59 in 2 lines

Reply:
Thanks for the comments,  updated it.